src/ng-generate/components/shared/generators/language/index.ts

```diff
@@ -79,6 +79,11 @@ function getBlockTransEntity(property: Property, lang: string): string {
 
 function getBlockEntityInstance(property: Property, lang: string, parentPropertyName = ''): string {
     const {characteristic} = property;
+
+    if (characteristic instanceof DefaultEnumeration && characteristic.values[0] instanceof DefaultEntityInstance) {
```

Comment:
Check values are there otherwise there can be an error: characteristic.values?.[0]